### PR TITLE
Add next/prev page, jump to start/end to user config

### LINKED ISSUE #482: Request: Custom keybinding for pagination (scroll up, down, start, end)
As far as I can tell, the list of actions which can be bound to keys by the user are found [here](https://github.com/Rigellute/spotify-tui/blob/master/src/user_config.rs#L227). I'd like to be able to bind my `pageup`, `pagedown`, `home`, and `end` keys to: 

- "Scroll up to previous result page"
- "Scroll down to next result page"
- "Jump to start of playlist"
- "Jump to end of playlist"

Thanks for all work on this project!

### PR DESCRIPTION
**Pull request by Toaster192**
resolves #482 
@AndydeCleyre, @vidurb might want to know

This allows the 4 keybinds to be bound to something else (something that doesn't colide with people's tmux keybinds and such)

### CODE REVIEW

File: src/handlers/track_table.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ pub fn handler(key: Key, app: &mut App) {
       on_enter(app);
     }
     // Scroll down
-    Key::Ctrl('d') => {
+    k if k == app.user_config.keys.previous_page => {
```
Comment (Rigellute) [inline src/handlers/track_table.rs:43]: Looks like this should be `next_page`

File: src/handlers/track_table.rs [hunk 2]
```diff
@@ -93,7 +93,7 @@ pub fn handler(key: Key, app: &mut App) {
       };
     }
     // Scroll up
-    Key::Ctrl('u') => {
+    k if k == app.user_config.keys.next_page => {
```
Comment (Rigellute) [inline src/handlers/track_table.rs:96]: And this should be `previous_page`

**Review by Rigellute — COMMENTED** [review]
This should be a great addition. Thank you for your work on this. 

Looks like we'd want to also update the handlers in these files, to use the new next/prev page config
```
src/handlers/home.rs
src/handlers/album_list.rs
```

**Comment by vidurb** [comment]
Thanks very much, @Toaster192

Your efforts are much appreciated.

**Comment by Toaster192** [comment]
@Rigellute Good catch, thanks!
@vidurb Thanks for the kind words!
I've literally never before worked with Rust so please excuse me if there are any stupid mistakes (tho it's just a few lines of code)

**Review by Rigellute — APPROVED** [review]
Thank you! 👏

**Comment by Rigellute** [comment]
@all-contributors please add @Toaster192 for code

**Pull request merged**